docs/quickstart.rst

```diff
@@ -33,19 +33,26 @@ there is both synchronous and asynchronous API available.
 
 You can see all Full Node Client's methods :ref:`FullNodeClient`.
 
-Creating AccountClient
+Creating Account
 ----------------------
 
-:obj:`AccountClient <starknet_py.net.account.account_client.AccountClient>` is an extension of a regular client. It leverages `OpenZeppelin's Cairo contracts <https://github.com/OpenZeppelin/cairo-contracts>`_ to create an account contract which proxies (and signs) the calls to other contracts on Starknet.
+.. warning::
 
-AccountClient can be created in two ways:
+    AccountClient has been deprecated in favor of new Account.
+    New Account doesn't implement a Client interface in favor of composition.
+    We recommend migrating to new Account as AccountClient will be removed in the future.
+
+:obj:`Account <starknet_py.net.account.account.Account>` is a default implementation of :obj:`BaseAccount <starknet_py.net.account.base_account.BaseAccount>` interface.
+It supports an account contract which proxies (and signs) the calls to other contracts on StarkNet.
```

Comment:
"(and signs)" should be removed from this context, contract doesn't sign anything.